packages/server/modules/core/services/users.js

```diff
@@ -27,6 +27,8 @@ const _ensureAtleastOneAdminRemains = async ( userId ) => {
   }
 }
 
+const userByEmailQuery = email => Users( ).where( 'email', 'ilike', `%${email}` )
```

Comment:
You might want to consider using the `lower` function rather than `like` here. It's much more performant and this query gets called a lot. https://www.postgresql.org/docs/current/indexes-expressional.html

Reply:
@gjedlicska

Reply:
Might be worth creating an index on the column as well `CREATE UNIQUE INDEX lower_email_idx ON email ((lower(email)));`

Reply:
yes, I know, the reason why it is there, is not using case insensitive compare, breaks older server installations.

So previously we would save the user email as entered like `eXamPle@eMaIL.AddreSs`.
Given this, or the previous implementation, every time we would try to lookup the user with `example@email.address` making it so, that an existing user would be locked out of their valid account.

Reply:
Then in the lookup you use `SELECT username FROM users WHERE lower(email) = 'MiXeD.Case@gmail.com';` with no performance hit